arwen/contexts/runtime.go

```diff
@@ -42,6 +44,12 @@ type runtimeContext struct {
 	errors          arwen.WrappableError
 }
 
+type gasTracer struct {
+	traceGasEnabled           bool
+	gasTrace                  map[string]map[string][]uint64
```

Comment:
Consider defining a type alias for `map[string]map[string][]uint64`, e.g. a type called `GasTraceMap`.